src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,15 @@
 // your components will all go in this `component` directory.
 // feel free to change this component.js into TodoList.js
+import React, { Component } from 'react';
+
+class TodoList extends Component {
+    render(){
+       return(
+        <div className="container-ListItems">
+        {this.props.tasks.map( (x,i) =><div className={`list-item ${x.completed ? "item-complete" :""}`} key={i} onClick={() => this.props.cb_click(i)}>{x.task}</div>
+        )}
+       </div>
+       )
```

Comment:
For your map keys, you could also use the unique task ID's (especially if you change them to be even more randomized than the current date conversion method) as [recommended in these cases by official React documentation](https://reactjs.org/docs/lists-and-keys.html#keys).

Also, you could name the item parameter something other than `x` (perhaps you could call it `task` - singular form of the array name).